docs: add info for babel7 peer dep (#173)
vuejs/vue-jest: README.md

npm install --save-dev vue-jest
```

### Usage with Babel 7

> If you use the latest Version of [jest](https://github.com/facebook/jest) and [babel-jest](https://github.com/facebook/jest/tree/master/packages/babel-jest) make sure to install the needed peer-dependency of `vue-jest`

Reviewer: ```suggestion
If you use the latest Version of [jest](https://github.com/facebook/jest) and [babel-jest](https://github.com/facebook/jest/tree/master/packages/babel-jest) make sure to install babel-core bridge
```

> If you use the latest Version of [jest](https://github.com/facebook/jest) and [babel-jest](https://github.com/facebook/jest/tree/master/packages/babel-jest) make sure to install the needed peer-dependency of `vue-jest`

Reviewer: ```suggestion
If you use the latest version of [jest](https://github.com/facebook/jest) and [babel-jest](https://github.com/facebook/jest/tree/master/packages/babel-jest) make sure to install the needed peer-dependency of `vue-jest`
```

```bash

> npm install --save-dev babel-core@7.0.0-bridge.0

Reviewer 2: use tag: `npm install --save-dev babel-core@bridge`

Reviewer 3: @donaldshen saved my life.

> npm install --save-dev babel-core@7.0.0-bridge.0

Reviewer: ```suggestion
npm install --save-dev babel-core@bridge
```

### Usage with Babel 7

> If you use [jest](https://github.com/facebook/jest) < 24.0.0 and [babel-jest](https://github.com/facebook/jest/tree/master/packages/babel-jest) make sure to install babel-core@bridge

Reviewer 4: It should say > 24.0.0

Reviewer 4: Confused me quite a lot
